Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/JavaArrayList.java

```diff
@@ -24,19 +24,19 @@ public static void main(String[] args) {
 
     }
 
-    private static ArrayList<ArrayList<Integer>> createTable(int rowsNumber) {
-        ArrayList<ArrayList<Integer>> table = new ArrayList<>(rowsNumber);
+    private static List<List<Integer>> createTable(int rowsNumber) {
+        List<List<Integer>> table = new ArrayList<>(rowsNumber);
         for (int rowIndex = 0; rowIndex < rowsNumber; rowIndex++) {
             int numbersInRow = scanner.nextInt();
-            ArrayList<Integer> row = createRow(numbersInRow);
+            List<Integer> row = createRow(numbersInRow);
             table.add(row);
         }
         scanner.nextLine();
         return table;
     }
 
-    private static ArrayList<Integer> createRow(int numbersInRow) {
-        ArrayList<Integer> row = new ArrayList<>(numbersInRow);
+    private static List<Integer> createRow(int numbersInRow) {
+        List<Integer> row = new ArrayList<>(numbersInRow);
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...